index.html

```diff
@@ -5052,6 +4769,366 @@ <h3>Cleaning up the data</h3>
+					<tr>
+						<td>
+							<code>duration</code>
+						</td>
+						<td>
+							<a href="#duration"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>editor</code>
+						</td>
+						<td>
+							<a href="#creators"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>id</code>
+						</td>
+						<td></td>
+						<td><a href="#canonical-identifier"></a></td>
+					</tr>
+					<tr>
+						<td>
+							<code>illustrator</code>
+						</td>
+						<td>
+							<a href="#creators"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>inDirection</code>
+						</td>
+						<td>
+							<a href="#language-and-dir"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>inker</code>
+						</td>
+						<td>
+							<a href="#creators"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>inLanguage</code>
+						</td>
+						<td>
+							<a href="#language-and-dir"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>letterer</code>
+						</td>
+						<td>
+							<a href="#creators"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>link</code>
+						</td>
+						<td>
+							<a href="#links"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>name</code>
+						</td>
+						<td>
+							<a href="#pub-title"></a>
+						</td>
+						<td> Extended in <a href="#wp-title"></a>
+						</td>
+					</tr>
+					<tr>
+						<td>
+							<code>penciler</code>
+						</td>
+						<td>
+							<a href="#creators"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>publisher</code>
+						</td>
+						<td>
+							<a href="#creators"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>readBy</code>
+						</td>
+						<td>
+							<a href="#creators"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>readingOrder</code>
+						</td>
+						<td>
+							<a href="#default-reading-order"></a>
+						</td>
+						<td> Extended in <a href="#wp-reading-order"></a>
+						</td>
+					</tr>
+					<tr>
+						<td>
+							<code>readingProgression</code>
+						</td>
+						<td>
+							<a href="#reading-progression-direction"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>resources</code>
+						</td>
+						<td>
+							<a href="#resource-list"></a>
+						</td>
+						<td></td>
+					</tr>
+					<tr>
+						<td>
+							<code>translator</code>
+						</td>
+						<td>
+							<a href="#creators"></a>
+						</td>
+						<td></td>
+					</tr>
+				</tbody>
+			</table>
+		</section>
+		<section id="rel-index" class="appendix informative">
+			<h4>Resource Relations Index</h4>
+
+			<p>The following table lists where the resource relations are defined in this specification.</p>
+
```

Comment:
This is just me being picky: we do _define_ these rel values _except_ for `contents`, which is a registered IANA term.

Reply:
described?

Reply:
In the interests of dodging the problem, I've changed the wording to make "the use of" what is defined - "The following table identifies where the use of resource relations is defined."